### PR TITLE
process for upgrade

### PR DESCRIPTION
**Pull request by itdove**
Signed-off-by: Dominique Vernier <dvernier@redhat.com>

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -40,4 +40,6 @@ kind_kubeconfig.yaml
 kind_kubeconfig_internal.yaml
 
 /operator-sdk
-/yq
\ No newline at end of file
+/yq
+
+# bundle/manifests/idp-mgmt-operator.clusterserviceversion.yaml
```
Comment (vidyanambiar) [inline .gitignore:45]: Is this line required in .gitignore and if so, can we uncomment it?

File: bundle/manifests/idp-mgmt-operator.clusterserviceversion.yaml [hunk 2]
```diff
@@ -65,6 +73,7 @@ spec:
         - apiGroups:
           - ""
           resources:
+          - namespaces
           - pods
```
Comment (vidyanambiar) [inline bundle/manifests/idp-mgmt-operator.clusterserviceversion.yaml:77]: Why is the bundle manifest committed?

File: .gitignore (hunk 1 above)
Comment (itdove) [reply .gitignore:45]: We will need it later when downstream will generate it.

File: bundle/manifests/idp-mgmt-operator.clusterserviceversion.yaml (hunk 2 above)
Comment (chrisahl) [reply bundle/manifests/idp-mgmt-operator.clusterserviceversion.yaml:77]: This needs to be generated and committed based on what Andrew and Gurney told us.  It is not generated during downstream build (just select fields are replaced, like version)

Comment (itdove) [reply bundle/manifests/idp-mgmt-operator.clusterserviceversion.yaml:77]: it already was.
The only change is when I added the check if ACM/MCE already exists I added a role in the installer (probably that role was already present as it was working) but I didn't regenerate that file.

File: .gitignore (hunk 1 above)
Comment (chrisahl) [reply .gitignore:45]: We can't ignore this file

**Review by vidyanambiar — APPROVED** [review]
/lgtm

File: .gitignore (hunk 1 above)
Comment (itdove) [reply .gitignore:45]: I'll add the new line but it wasn't there before.

**Pull request merged**